alpine-rails-buildkite/Dockerfile

```diff
@@ -0,0 +1,32 @@
+FROM dkubb/alpine-rails-nginx
+
+# Improve root state, intend to be pushed there
+WORKDIR /
```

Comment:
The parent container leaves the `WORKDIR` set to: `/opt/rails` a path other users cannot access.

Conceptually I think we should make a parent container *both* the rails one and the buildkite one inherit from.

Hence I did not pushed this setting to the current parent container and the comment applies to the "planned" real-parent container.